src/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js

```diff
@@ -25,32 +25,41 @@ const useStyles = makeStyles(theme => ({
 const DashboardPlaceholder = (props) => {
   const classes = useStyles();
   const { t } = useTranslation();
-  const { labelKey, defaultLabel, icon } = props;
+  const { title, label, icon } = props;
 
   return (
-    <Grid container justify="center" alignItems="center" className={classes.gridContainer}>
+    <Grid container spacing={2} direction="column" justify="center" wrap="nowrap"
+      alignItems="center" className={classes.gridContainer}>
+      <Grid>
+        <Typography variant="h2" >
+          {title}
+        </Typography>
+      </Grid>
       <Grid item>
         { icon !== undefined &&
           <div className={classes.iconContainer}>
             {icon}
           </div>
         }
         <Typography
-          component="h2"
```

Comment:
Why did you remove the h2 component?

Reply:
Beacause label case should be different from title case